### DISCUSSION TITLE
Error: "No Compatible Engine Installed" when trying to create a Landscape Mountains project

### DISCUSSION THREAD

**Discussion by LeonardoRez** [microsoft/AirSim#3892] [Support Q&A]
Hello. I'm trying to follow the steps on [Creating and Setting Up Unreal Environment](https://github.com/microsoft/AirSim/blob/master/docs/unreal_custenv.md) , but although the step 1 says to make sure Unreal 4.25 is installed, the Landscape Mountains page on marketplace says that it's only supported by 4.4-4.24.

![image](https://user-images.githubusercontent.com/13459436/126229507-1885db1e-2678-4fb7-a75d-9a991503fc34.png)

When I hit "create project", the most recent version available is 4.24. I tried to download it and create a project ignoring the alert, but it doesn't work.

Is there some configuration I'm missing? This landscape is supposed to work with UE 4.25?

Thank you very much

**Comment by xxEoD2242** [comment]
If you build it in 4.24 then compile again manually in 4.25 it will work
